SYCL/lit.cfg.py

```diff
@@ -200,8 +200,12 @@
     fp.write('int main() {}')
 
 extra_sycl_include = ""
-sycl_hpp_available = subprocess.getstatusoutput(config.dpcpp_compiler+' -fsycl  ' + check_sycl_hpp_file)
+sycl_hpp_available = subprocess.getstatusoutput(('env LIB="' if platform.system() == "Windows" else 'env LD_LIBRARY_PATH=')
```

Comment:
the change will not wor out of unix-like console on windows.
I suggest to do a compilation only as it is enough to check include presence.